pkg/manifests/manifests.go

```diff
@@ -847,6 +854,13 @@ func (f *Factory) updateNodeExporterArgs(args []string) []string {
 		if f.config.ClusterMonitoringConfiguration.NodeExporterConfig.Collectors.NetClass.UseNetlink {
 			args = setArg(args, "--collector.netclass.netlink", "")
 		}
+		excludedDev := *f.config.ClusterMonitoringConfiguration.NodeExporterConfig.Collectors.NetClass.IgnoredNetworkDevices
+		if len(excludedDev) > 0 {
+			args = setArg(args, "--collector.netclass.ignored-devices=", regexListToArg(excludedDev))
+		} else {
+			// make sure upstream default isn't used
```

Comment:
To keep it simple, I'd configure node_exporter with `--collector.netclass.ignored-devices=""` if the list is empty (it removes 1 if branch).